tests/ui/apps/ArcPicker/ArcPicker-View.js

```diff
@@ -0,0 +1,18 @@
+import ArcPicker from '../../../../ArcPicker';
+import ThemeDecorator from '../../../../ThemeDecorator';
+import React from 'react';
+import spotlight from '@enact/spotlight';
+
+// NOTE: Forcing pointer mode off so we can be sure that regardless of webOS pointer mode the app
+// runs the same way
+spotlight.setPointerMode(false);
+
+const app = (props) => <div {...props}>
+	<div>
+		<ArcPicker id="arcPickerDefault">{[1, 2, 3, 4]}</ArcPicker>
```

Comment:
It would be a good idea we are adding titles for each ArcPicker to distinguish which one is which. We could use the Heading component or something else.